### PR TITLE
See more

### PR DESCRIPTION
**Pull request by rairohit17**
## Description
This pull request enhances the verse similarity feature in our application by implementing see-more functionality, allowing users to access a broader range of similar verses. The following changes have been made:

[Screencast from 2024-10-19 04-04-22.webm](https://github.com/user-attachments/assets/bb82764a-9ecc-4845-8fa3-b87f12fce8a2)

I apologize for the delay in submitting this pull request. There were some bugs in the backend that hindered the frontend's ability to communicate effectively with the backend, which required resolution before proceeding with these enhancements.

### CODE REVIEW

**Comment by JustinhSE** [comment]
How did you even know that scripture would trigger that 😂. But functionality-wise, it looks good.

File: backend/routes/verseSearch.py [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+from flask_cors import CORS  # Import CORS
 from utils.utils import get_similar_verses
 
+
+
+
 bp = Blueprint('verseSearch', __name__)
+CORS(bp, resources={r"/api/*": {"origins": "http://localhost:3000"}})
+
+# Enable CORS for the blueprint (you can also apply it to the app)
```
Comment (JustinhSE) [inline backend/routes/verseSearch.py:11]: @Namit2111 are we adding CORS to our project?

File: backend/routes/__pycache__/__init__.cpython-312.pyc
Comment (JustinhSE) [inline backend/routes/__pycache__/__init__.cpython-312.pyc:1]: why are you pushing binary files? @rairohit17

**Comment by rairohit17** [comment]
I’ve removed the mistakenly added binary files and cleaned up the branch. The changes should now be ready for review.

Please let me know  @JustinhSE if any further adjustments are needed. Thanks!

**Comment by Namit2111** [comment]
HI @rairohit17 please resolve the conflicts remove all the binary files and remove cors as of now you are using localhost in cors that is unecessary in the production. Solve these issues and create another PR also squash all the commits while making a PR

**Comment by JustinhSE** [comment]
After removing bin files and cors, run `run "npm run build` it will show all the build issues. Resolve these as well

**Pull request closed without merging**